src/ui/KonvergoWindow.cpp

```diff
@@ -773,3 +774,22 @@ void KonvergoWindow::updateCurrentScreen()
   if (currentName != m_currentScreenName)
     updateScreens();
 }
+
+/////////////////////////////////////////////////////////////////////////////////////////
+void KonvergoWindow::setWindowFullScreen(bool fullScreen)
+{
+#ifdef Q_OS_MAC
+    if (fullScreen) {
```

Comment:
You can probably make this block simpler with 
```
        setVisibility(QWindow::Windowed);
        OSXUtils::SetWindowFullScreen(this, fullscreen);
```

Reply:
New lines for brackets again.